ts/src/coinlist.ts

```diff
@@ -0,0 +1,2246 @@
+         * @returns {object} an list of [order structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
+         */
+        await this.loadMarkets ();
+        params = ids;
+        const response = await this.privateDeleteV1OrdersBulk (params);
+        return response;
+    }
+
+    async createOrder (symbol: string, type: OrderType, side: OrderSide, amount, price = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#createOrder
+         * @description create a trade order
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market' or 'limit' or 'stop_market' or 'stop_limit' or 'take_market' or 'take_limit'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float} [price] the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {object} an [order structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'type': type,
+            'side': side,
+            'size': this.numberToString (this.amountToPrecision (symbol, amount)),
+        };
+        if ((type === 'limit') || (type === 'stop_limit') || (type === 'take_limit')) {
+            if (price === undefined) {
+                throw new ArgumentsRequired (this.id + ' createOrder() requires a price argument for a ' + type + ' order');
+            }
+            request['price'] = this.priceToPrecision (symbol, price).toString ();
+        }
+        const postOnly = this.safeValue (params, 'postOnly', false);
+        if (postOnly) {
+            request['post_only'] = true;
+        }
+        const triggerPrice = this.safeNumberN (params, [ 'triggerPrice', 'stopPrice', 'stop_price' ]);
+        if (triggerPrice !== undefined) {
+            request['stop_price'] = this.priceToPrecision (symbol, triggerPrice).toString ();
+            if (type === 'market') {
+                request['type'] = 'stop_market';
+            } else if (type === 'limit') {
+                request['type'] = 'stop_limit';
+            }
+        } else if ((type === 'stop_market') || (type === 'stop_limit') || (type === 'take_market') || (type === 'take_limit')) {
+            throw new ArgumentsRequired (this.id + ' createOrder() requires a stopPrice parameter for stop-loss and take-profit orders');
+        }
+        params = this.omit (params, 'triggerPrice', 'stopPrice', 'stop_price', 'postOnly');
+        const response = await this.privatePostV1Orders (this.extend (request, params));
```

Comment:
we're missing the clientorderId unification
![image](https://github.com/ccxt/ccxt/assets/43336371/93eaab46-a5f8-4384-828f-979840188edf)